src/main/java/core/Main.java

```diff
@@ -0,0 +1,13 @@
+package core;
+
+import core.strategy.ReportFactory;
+
+public class Main {
+
+    public static void main(String[] args) {
```

Comment:
Your program should have a good structure and services name 
here is a diagram that should help you 
<img width="829" alt="image" src="https://github.com/mate-academy/jv-fruit-shop/assets/32339836/6544df25-2b9a-4df2-9665-4941feec347b">

you don't use SOLID in your solution. Fix it. Every service should have an interface and implementation and should be called from the main (see common mistakes carefully)


Be careful with PR names 
<img width="1027" alt="image" src="https://github.com/mate-academy/jv-fruit-shop/assets/32339836/1fa58b88-b041-4c71-b412-7c16d9fd4005">
"doned"  is the wrong word.